stdlib/grpc/src/main/resources/templates/skeleton/message.mustache

```diff
@@ -9,5 +9,4 @@ public type {{messageName}} record {|
 {{#if oneofFieldMap}}
 {{> oneoffield}}
 {{/if}}{{#enumList}}
-{{> enum}}{{/enumList}}{{#if this.nestedMessageList}}{{#each this.nestedMessageList}}
-{{> message}}{{/each}}{{/if}}
```

Comment:
Why we remove `nestedMessageList` from this template. Are we handling it else where

Reply:
The overall messageMap was changed to include all the nested message types as well (recursively built). So there is not need to read nested messages here.
I needed to include the nested messages as well in the overall message Map in order to read nested message values in query parameters, so I made this change.